src/init.cpp

```diff
@@ -744,7 +744,7 @@ static void LoadSaplingParams()
         uiInterface.ThreadSafeMessageBox(strprintf(
                 _("Cannot find the Sapling parameters in the following directory:\n"
                   "%s\n"
-                  "Please run 'sapling-fetch-params' or './util/fetch-params.sh' and then restart."),
+                  "Please run 'sapling-fetch-params' or the included install-params.sh and then restart."),
```

Comment:
In order to cover both Windows zip based installs, as well as linux/macOS tarball installs, we should probably rewrite this entire section to be more modular. Here is a diff patch based on current `master` that splits up the string into multiple individual strings (with string replacement variables), then uses pre-compiler macros to craft the finalized UI message that the user will see.

```diff
Index: src/init.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/init.cpp b/src/init.cpp
--- a/src/init.cpp	(revision f07f32c09a99542f292154051fadd13c2cdee40f)
+++ b/src/init.cpp	(date 1682153190539)
@@ -741,11 +741,15 @@
     try {
         initZKSNARKS();
     } catch (std::runtime_error &e) {
-        uiInterface.ThreadSafeMessageBox(strprintf(
-                _("Cannot find the Sapling parameters in the following directory:\n"
-                  "%s\n"
-                  "Please run 'sapling-fetch-params' or './util/fetch-params.sh' and then restart."),
-                ZC_GetParamsDir()),
+        std::string strError = strprintf(_("Cannot find the Sapling parameters in the following directory:\n%s"), ZC_GetParamsDir());
+        std::string strErrorPosix = strprintf(_("Please run the included %s script and then restart."), "install-params.sh");
+        std::string strErrorWin = strprintf(_("Please copy the included params files to the %s directory."), ZC_GetParamsDir());
+        uiInterface.ThreadSafeMessageBox(strError + "\n"
+#ifndef WIN32
+                  + strErrorPosix,
+#else
+                  + strErrorWin,
+#endif
                                          "", CClientUIInterface::MSG_ERROR);
         StartShutdown();
         return;

```